### DISCUSSION TITLE
Section headline for Other meetings at this location

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1535] [Ideas]
This might just be me, but when there are multiple meetings at one location, there is not subhead to make that clear what that additional listing is. Is there a way to create a headline that says **Other meetings at this location** before the additional meeting list?
Thanks
Rose L
District 6 NC

**Comment by joshreisner** [comment]
hi @district6wc to confirm, is the place you're talking about?

<img width="728" alt="Screenshot 2024-10-02 at 9 47 52 AM" src="https://github.com/user-attachments/assets/1c31893a-7205-4f98-8281-e94ba6b9eab3">

**Comment by district6wc** [comment]
Yes, I think if there was a headline there it would be clearer.


Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Comment by joshreisner** [comment]
hm, this lists all the meetings at that location, including the one you're looking at, so we'd need either to change the behavior and exclude the one you're looking at, or change the headline - maybe `{count} Meetings at this Location`

perhaps this should be an accordion and not show the individual meetings by default, because some of these get a little long: https://aasanjose.org/meetings?meeting=many-do-recover

we'd also need to do this in the group section, maybe `{count} Meetings in this Group` or `by this Group`? not really sure what preposition to use for that

**Comment by district6wc** [comment]
JOsh

I was just curious. I have some not very tech confident people in our
district so I try to be as clear about everything as I can be. It was just
a suggestion at this point, I am sure a s people get used to the new
interface it will be clearer for them.
Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328
